src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
 // write your code here
+const getLogo = document.querySelector('.logo');
+const createDiv = document.createElement('div');
+
+createDiv.classList.add('message');
+
+const promise1 = new Promise((resolve) => {
+  getLogo.addEventListener('click', () => {
+    if (resolve) {
+      resolve('Promise was resolved!');
+    }
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  if (reject) {
```

Comment:
The condition `if (reject)` is unnecessary because `reject` is always a function provided by the Promise constructor. You can directly call `reject(new Error('Promise was rejected!'))`.